### ISSUE TITLE
scss error?

### ISSUE THREAD

**Issue by kylegoines** [zacky1972/gulp-parcel#10]
Hello I seem to be throwing errors even though compliation seems to be happending correctly

this:
```
gulp.src('_scss/main.scss', {read:false})
            .pipe( parcel( { outDir: 'dist' } ) )
            .pipe( gulp.dest('dist') )
```
compiles and works with scss but after hanging for a second i seem to be getting an unhanded promise error

```
UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): [object Object]
...
/node_modules/gulp-parcel/index.js:112
						throw err;
						^
```

**Comment by kylegoines** [comment]
to clarify, this is comping to css correctly but is halting my watch task.
any help would be great :)

**Comment by a65162** [comment]
same problem here

**Comment by davepeck** [comment]
After looking at the code behind `gulp-parcel`, this just seems to be completely busted.

Basically, `gulp-parcel` incorrectly computes `out_flname` -- it assumes that the output filename has the same extension as the input: for instance, if you start with `main.js`, you'll end with `main.js`. You can see the [computation of `out_flname` here](https://github.com/zacky1972/gulp-parcel/blob/master/index.js#L64).

At the same time, it's clear that the author recognized this issue, but not in the general case: if you look further below [in `gulp-parcel`'s `index.js`](https://github.com/zacky1972/gulp-parcel/blob/master/index.js#L103) you can see where the code explicitly looks for the `.pug` extension in the input file name, and rewrites the output file name to `.html` so that no exception gets thrown.

Bottom line: we're not quite ready for prime time. Will have to look elsewhere or submit a PR...
